pro/src/new_components/ProfileForm/ProfileForm.tsx

```diff
@@ -0,0 +1,98 @@
+import { Button, SubmitButton } from 'ui-kit'
+import { Form, FormikProvider, useFormik } from 'formik'
+import React, { useState } from 'react'
+
+import { AnySchema } from 'yup'
+import { ButtonVariant } from 'ui-kit/Button/types'
+import FormLayout from 'new_components/FormLayout'
+import Icon from 'components/layout/Icon'
+import styles from './ProfileForm.module.scss'
+
+export interface IProfileFormProps {
+  title: string
+  value: string
+  fields: Array<JSX.Element>
+  validationSchema: AnySchema
+  banner?: JSX.Element
+  shouldDisplayBanner: boolean
+  onSubmit: (values: any) => void
+  initialValues: { [id: string]: string }
+}
+
+const ProfileForm = ({
+  title,
+  value,
+  fields,
+  validationSchema,
+  initialValues,
+  shouldDisplayBanner = false,
+  banner,
+  onSubmit,
+}: IProfileFormProps): JSX.Element => {
+  const [isFormVisible, setisFormVisible] = useState(false)
+  const toggleFormVisible = () => {
+    setisFormVisible(oldValue => !oldValue)
+  }
+  const formik = useFormik({
+    initialValues: initialValues,
+    onSubmit: onSubmit,
+    validationSchema,
+    validateOnChange: false,
+  })
+  return (
+    <div className={styles['profile-form']} data-testid="test-profile-form">
+      <div className={styles['profile-form-description']}>
+        <div className={styles['profile-form-description-column']}>
+          <div className={styles['profile-form-description-title']}>
+            {title}
+          </div>
+          <div className={styles['profile-form-description-value']}>
+            {value}
```

Comment:
Ca ne serait pas plutot : 
```suggestion
            {subtitle}
```
ou bien 
```suggestion
            {description}
```